recipes/libgomp/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "4.8.5" %}
+
+package:
+  name: libgomp
+  version: {{ version }}
+
+requirements:
+  build:
+    - gcc {{ version }}
+  run:
+    - libgcc_s {{ version }}    # [osx]
```

Comment:
Appears Linux does not have this dependency. Only OS X has it. Bit strange, but that's what `ldd`/`otool -L` tell us.